src/compiler/types.ts

```diff
@@ -1693,6 +1693,8 @@ namespace ts {
           * @param path The path to test.
           */
         fileExists(path: string): boolean;
+
+        readFile(path: string): string;

```

Comment:
@chuckjaz It looks like you have a readfile method available, you just dont pass it in as part of the host, so it looks easy to update for future versions. Actually, is there a reason you capture the methods on a new object, rather than just passing in `this`?